lib/versioning/loose/index.ts

```diff
@@ -50,6 +50,10 @@ class LooseVersioningApi extends GenericVersioningApi {
         return part1 - part2;
       }
     }
+    // istanbul ignore if
+    if (!(parsed1.suffix && parsed2.suffix)) {
+      return 1;
+    }
```

Comment:
@rarkins @viceice I think this is cause of Docker builds for swift

Reply:
No